cypress/integration/chi-custom-elements/brand.spec.js

```diff
@@ -0,0 +1,65 @@
+describe('Brand', () => {
+  it('Base brand should be included inside the custom element', () => {
+    cy.visit('tests/custom-elements/brand.html');
+
+    cy.get('[data-cy="brand-base-default"]', { timeout: 5000 })
+      .should('have.class', 'hydrated')
+      .children()
+      .first()
+      .should('match', 'a.a-brand')
+      .children()
+      .first()
+      .should('have.id', 'brand-base');
+  });
```

Comment:
I think all these steps could be wrapped as a function with just the id and data-cy as parameters.